aws/resource_aws_launch_template.go

```diff
@@ -277,34 +279,31 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 							Optional: true,
 						},
 						"security_groups": {
-							Type:     schema.TypeSet,
```

Comment:
These (`security_groups`, `ipv6_addresses`, and `ipv4_addresses`) will likely need to remain `Type: TypeSet` (just needed the `Set: schema.HashString` removed) as the API likely does not return them in a consistent order. We'll find out!